GPS playback example (#192)
farm-ng/amiga-dev-kit: website/docs/examples/camera_calibration/README.md

## Camera Calibration

> The requirement to run this example is to have a
> [**farm-ng brain**](/docs/brain/) running Oak cameras. This can be run either
> directly from a brain or from your local PC. If running from your local PC,
> ensure it's connected to the same local network as the brain
> or linked to it through tailscale.

Reviewer: To keep things more human readable in the markdown, always break lines on periods and try to break other lines near punctuation, when possible.

```suggestion
The requirement to run this example is to have a
[**farm-ng brain**](/docs/brain/) running Oak cameras.
This can be run either directly from a brain or from your local PC.
If running from your local PC,
ensure it's connected to the same local network as the brain or linked to it through tailscale.
```

```

:::info

> By default, the camera address is assumed to be `localhost` (service_config.json - line 4).

Reviewer: Use the field name rather than the line number (line numbers a re subject to change, field names should stay static)
```suggestion
By default, the camera host is assumed to be `localhost`.
```

If running the example from your local PC, you should replace `localhost` with your brain's

> IP address (e.g.,100.67.32.5) or your amiga's name (e.g., aluminum-pineapple -

Reviewer: ```suggestion
IP address (e.g., 100.67.32.5) or your Amiga's name (e.g., aluminum-pineapple -
```
